ament_cmake_test/cmake/run_test.py

```diff
@@ -42,6 +42,9 @@ def main(argv=sys.argv[1:]):
              'It must be passed after other arguments since it collects all '
              'following options.')
     parser.add_argument(
+        '--env',
+        help='extra environment to set when running, in the format FOO=foo;BAR=bar')
```

Comment:
I think a list of env arguments would be more intuitive instead of the semicolon separated one.